media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
And another question: why not using the shepherd.js ? http://github.hubspot.com/shepherd/docs/welcome/

This comes with only one dependency https://github.com/HubSpot/tether

Please use that script, we are trying to freed joomla from jQuery!!